tests/geomultiscale-mapping/DimensionsConfiguration.cpp

```diff
@@ -0,0 +1,92 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/Participant.hpp>
+#include <vector>
+#include "testing/TestContext.hpp"
+
+using namespace precice;
+using precice::testing::TestContext;
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(GeomultiscaleMapping)
+BOOST_AUTO_TEST_CASE(DimensionsConfiguration)
```

Comment:
This test could need more work (not sure yet to which direction). In principle, we could even delete it for now.

Reply:
Let's remove for the moment.